deepchecks/core/serialization/suite_result/html.py

```diff
@@ -196,7 +196,8 @@ def prepare_summary(self, output_id: t.Optional[str] = None, **kwargs) -> str:
         extra_info = self.prepare_extra_info()
 
         suite_creation_example_link = (
-            'https://docs.deepchecks.com/en/stable/examples/guides/create_a_custom_suite.html'
+            'https://docs.deepchecks.com/en/stable/general/usage/customizations/'
```

Comment:
without the "/en/stable" in all links

Reply:
(also please search and see that there aren't any /en/stable links left. should be only /stable)